metaflow/flowspec.py

```diff
@@ -100,6 +100,13 @@ def __init__(self, use_cli=True):
         self._graph = FlowGraph(self.__class__)
         self._steps = [getattr(self, node.name) for node in self._graph]
 
+        # This must be set before calling cli.main() below (or specifically, add_custom_parameters)
+        parameters.parameters = [
```

Comment:
changing the import above will just require `parameters` instead of `parameters.parameters` now...

Reply:
thus, this is fine too..